ecocode-rules-specifications/SonarQubeIntegration.md

```diff
@@ -0,0 +1,45 @@
+\newpage
```

Comment:
maybe, could you test this process in a particular language to verify if all described is ok

Reply:
This page was originally copied from: https://github.com/green-code-initiative/ecoCode-mobile-rules-specification/blob/main/rules/SonarQubeIntegration.md

However, it is premature to define the rules management process here. We need to discuss it before, and add this type of page once the organization has agreed together.

I will therefore remove this page from this PR.